src/FSharp.DependencyManager.Nuget/FSharp.DependencyManager.fs

```diff
@@ -12,93 +12,155 @@ open FSharp.DependencyManager.Nuget.ProjectFile
 open FSDependencyManager
 
 module FSharpDependencyManager =
-    
+
     [<assembly: DependencyManager>]
     do ()
 
-    let private concat (s:string) (v:string) : string =
+    let private concat (s: string) (v: string) : string =
         match String.IsNullOrEmpty(s), String.IsNullOrEmpty(v) with
         | false, false -> s + ";" + v
         | false, true -> s
         | true, false -> v
-        | _  -> ""
+        | _ -> ""
 
-    let validateAndFormatRestoreSources (sources:string) = [|
+    let validateAndFormatRestoreSources (sources: string) =
+        [|
             let items = sources.Split(';')
+
             for item in items do
                 let uri = Uri(item)
+
                 if uri.IsFile then
                     let directoryName = uri.LocalPath
+
                     if Directory.Exists(directoryName) then
-                        yield sprintf """  <PropertyGroup Condition="Exists('%s')"><RestoreAdditionalProjectSources>$(RestoreAdditionalProjectSources);%s</RestoreAdditionalProjectSources></PropertyGroup>""" directoryName directoryName
+                        yield
+                            sprintf
+                                """  <PropertyGroup Condition="Exists('%s')"><RestoreAdditionalProjectSources>$(RestoreAdditionalProjectSources);%s</RestoreAdditionalProjectSources></PropertyGroup>"""
+                                directoryName
+                                directoryName
                     else
-                        raise (Exception(SR.sourceDirectoryDoesntExist(directoryName)))
+                        raise (Exception(SR.sourceDirectoryDoesntExist (directoryName)))
                 else
-                    yield sprintf """  <PropertyGroup><RestoreAdditionalProjectSources>$(RestoreAdditionalProjectSources);%s</RestoreAdditionalProjectSources></PropertyGroup>""" uri.OriginalString 
+                    yield
+                        sprintf
+                            """  <PropertyGroup><RestoreAdditionalProjectSources>$(RestoreAdditionalProjectSources);%s</RestoreAdditionalProjectSources></PropertyGroup>"""
+                            uri.OriginalString
         |]
 
     let formatPackageReference p =
-        let { Include=inc; Version=ver; RestoreSources=src; Script=script } = p
+        let {
+                Include = inc
+                Version = ver
+                RestoreSources = src
+                Script = script
+            } =
+            p
```

Comment:
These expansions of pattern matching on records are a really good reason not to use pattern matching on records! (I generally don't like the use of the feature).